peps/pep-0763.rst

```diff
@@ -0,0 +1,269 @@
+PEP: 763
+Title: Limiting deletions on PyPI
+Author: William Woodruff <william@yossarian.net>,
+        Alexis Challande <alexis.challande@trailofbits.com>
+Sponsor: Donald Stufft <donald@stufft.io>
+PEP-Delegate: Donald Stufft <donald@stufft.io>
+Status: Draft
+Type: Standards Track
+Topic: Packaging
+Created: 07-Oct-2024
```

Comment:
One last thing :)  This is the PEP number assignment date.
```suggestion
Created: 24-Oct-2024
```

Reply:
Fixed, thanks!